frontend/src/reducers/cartReducers.js

```diff
@@ -0,0 +1,28 @@
+import { CART_ADD_ITEM, CART_REMOVE_ITEM } from "../constants/cartConstants";
+
+export const cartReducer = (state = { cartItems: [] }, action) => {
+    switch (action.type) {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Durotemy/e-commerce/pullRequest?prid=9481957)
```suggestion
  switch (action.type) {
```